testsuite/features/secondary/srv_menu.feature

```diff
@@ -149,8 +149,8 @@ Feature: Web UI - Main landing page menu, texts and links
 
   Scenario: The menu direct link accesses the first submenu level only
     When I follow the left menu "Patches > Patch List"
-    Then I should see a "All Types" text in the content area
-    And I should not see a "Patches Relevant to Your Systems" text in the content area
+    Then I should see a "Patches Relevant to Your Systems" text in the content area
+    And I should not see a "The following patch list represents all patches accessible by your organization." text in the content area
```

Comment:
Note: I could not just put "All Types" here since it's also the name of one of the tabs.